app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,160 @@
+                  </ol>
+                </td>
+              </tr>
+            </tbody>
+          </table>   
+        </div>
+      </div>
+  </div>
+  <div class="accord">
+    <div class="accord_header" id="headingTwo">
+      <h5 class="mb-0">
+        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">Plain Text Files
+        </button>
+      </h5>
+    </div>
+    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+      <div class="accord_body">
+        <p class="about_p">Each website within the web archive collection will have its full text presented on one line, along with information around when it was crawled, the name of the domain, and the full URL of the content. AUK users will have access to the Full Text file which contains the extracted plain text of all the HTML files found within a collection.</p>
+        <table id="accord_table">
+          <tbody class="tbody_table">
+            <tr class="tr_table">
+              <td class="td_table"><strong>Tutorial</strong></td>
+              <td class="td_table">
+                <ul>
+                  <li class="card_li"><strong><%=link_to("Text Analysis Part One: Beyond the Keyword Search: Using AntConc", "/derivatives/text-antconc")%></strong>: This tutorial explores how you can explore text within a web archive using the AntConc tool.</li>
+                  <li class="card_li"><strong><%=link_to("Text Analysis Part Two: Sentiment Analysis With the Natural Language Toolkit", "/derivatives/text-sentiment")%></strong>: This tutorial explores how you can calculate the positivity or negativity (in an emotional sense) of web archive text.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>File Output</strong></td>
+              <td class="td_table"><kbd>.txt</kbd></td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>Files Generated</strong></td>
+              <td class="td_table">A <strong>text file</strong> is generated which can be used in various other programs and software.</td>
+            </tr>
+            <tr class="tr_table">
+              <td class="td_table"><strong>Additional Resources</strong></td>
+              <td class="td_table">
+                <p class="about_p">Working with lots of text can be challenging. Fortunately, there are lots of resouces on the web to help teach you the basics of text analysis.</p>
+                <ul>
+                  <li class="card_li"><strong><a href="https://programminghistorian.org">Programming Historian</a></strong>: The Programming Historian has many lessons, including a specific section on <a href="https://programminghistorian.org/en/lessons/?topic=distant-reading">distant reading."</a></li>
```

Comment:
The text line spacing on the `<ul>` seems broader than the rest of the text, at least in my browser.

<img width="338" alt="screen shot 2018-08-10 at 8 55 30 pm" src="https://user-images.githubusercontent.com/3834704/43986510-bb80ef54-9cdf-11e8-8eec-240f1951821d.png">